scripts/install-mender.sh

```diff
@@ -266,12 +281,48 @@ do_install_commercial() {
     echo "  Success!"
 }
 
+do_setup_mender() {
+    # Return if mender-client was not installed
+    if [[ ! "$SELECTED_COMPONENTS" == *"mender-client"* ]]; then
+        return
+    fi
+
+    # Return if no setup options were passed
+    if [ -z "$MENDER_SETUP_ARGS" ]; then
+        return
+    fi
+
+    echo "  Setting up mender with options: $MENDER_SETUP_ARGS"
+    mender setup $MENDER_SETUP_ARGS
+    pidof systemd && systemctl restart mender-client
+    echo "  Success!"
+}
+
+do_setup_addons() {
+    # Setup for mender-connect, hardcoded for Raspbian OS
+    if [[ "$SELECTED_COMPONENTS" == *"mender-connect"* ]]; then
+        if [ "$DEMO" -eq 1 ]; then
+            echo "  Setting up mender-connect for demo"
+            cat > /etc/mender/mender-connect.conf << EOF
+{
+  "User": "pi",
```

Comment:
I know you said it's hardcoded for RPi, but I'd say that "root" is a better default overall, at least for demo. It will work everywhere. Or is there a specific reason we should use "pi"?

Reply:
I have no good reason, maybe `root` is a good idea too. I used `pi` basically because we had it [in the GUI code](https://github.com/mendersoftware/gui/pull/1978/files#diff-68dd097fe57104078af21614d0402539c2377c35e2f44881225afff6a555a751L2011).

Let's see if others here have opinions, otherwise I suggest changing it to `root` but adding a log message here saying something like `Configuring mender-connect for user "root" and shell "bash"...`

Reply:
Done